Doc/c-api/float.rst

```diff
@@ -76,3 +76,83 @@ Floating Point Objects
 .. c:function:: double PyFloat_GetMin()
 
    Return the minimum normalized positive float *DBL_MIN* as C :c:type:`double`.
+
+
+Pack and Unpack functions
+=========================
+
+The :mod:`struct` and :mod:`pickle` (at least) modules need an efficient
+platform- independent way to store floating-point values as byte strings. The
+Pack routines produce a bytes string from a C :c:type:`double`, and the Unpack
+routines produce a C :c:type:`double` from such a bytes string. The suffix
+(4 or 8) specifies the number of bytes in the bytes string.
```

Comment:
2, 4, and 8?
Intentionally ignore 2?

Reply:
I copied the doc from the header file, I will add 2 once buildbot jobs complete.

Reply:
Fixed